sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/directconnectivity/rntbd/RntbdRequestRecord.java

```diff
@@ -48,29 +48,26 @@
             "stage");
 
     private final RntbdRequestArgs args;
-    private final RntbdRequestTimer timer;
 
     private volatile int requestLength;
     private volatile int responseLength;
     private volatile Stage stage;
 
     private volatile Instant timeCompleted;
     private volatile Instant timePipelined;
-    private volatile Instant timeQueued;
+    private final Instant timeQueued;
```

Comment:
why only timeQueued changed to final? should also change timeCompleted, timePipelined etc?